erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -141,6 +141,9 @@ def compute_error_tree_on_dataset(
     is_model_analyzer = hasattr(analyzer, MODEL)
     indexes = []
     for feature in features:
+        if feature not in analyzer.feature_names:
+            msg = 'Feature {} not found in dataset. Existing features: {}'
+            raise ValueError(msg.format(feature, analyzer.feature_names))
```

Comment:
Can we raise UserConfigValidationException instead of ValueError here. This should be available in the new raiutils package.

Reply:
done!